staging/src/k8s.io/apiextensions-apiserver/test/integration/validation_test.go

```diff
@@ -934,7 +946,9 @@ properties:
  metadata:
    type: object
 `,
-			expectedViolations: []string{},
+			expectedViolations: []string{
+				"spec.preserveUnknownFields: Invalid value: true: must be false",
```

Comment:
would instead set the value to false in all test fixtures by default, instead of changing all the expectations below.

Reply:
done!